ironfish-cli/src/commands/wallet/multisig/dkg/create.ts

```diff
@@ -0,0 +1,254 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { RpcClient } from '@ironfish/sdk'
+import { IronfishCommand } from '../../../../command'
+import { RemoteFlags } from '../../../../flags'
+import * as ui from '../../../../ui'
+
+export class DkgCreateCommand extends IronfishCommand {
+  static description = 'Interactive command to create a multisignature account using DKG'
+
+  static flags = {
+    ...RemoteFlags,
+  }
+
+  async start(): Promise<void> {
+    await this.parse(DkgCreateCommand)
+    const client = await this.connectRpc()
+    await ui.checkWalletUnlocked(client)
+
+    const name = await this.retryStep(async () => {
+      return this.getNameForMultisigAccount(client)
+    })
+
+    const { identity } = await this.retryStep(async () => {
+      return this.createParticipant(client, name)
+    })
+
+    const { round1Result, totalParticipants } = await this.retryStep(async () => {
+      return this.performRound1(client, name, identity)
+    })
+
+    const { round2Result, round1PublicPackages } = await this.retryStep(async () => {
+      return this.performRound2(client, name, round1Result, totalParticipants)
+    })
+
+    await this.retryStep(async () => {
+      await this.performRound3(
+        client,
+        name,
+        round2Result,
+        round1PublicPackages,
+        totalParticipants,
+      )
+    })
+
+    this.log('Multisig account created successfully using DKG!')
+  }
+
+  private async getNameForMultisigAccount(client: RpcClient): Promise<string> {
+    const name = await ui.inputPrompt(
+      'Enter a name for your account and participant identity',
+      true,
+    )
+
+    const identities = (await client.wallet.multisig.getIdentities()).content.identities
+
+    const accounts = (await client.wallet.getAccounts()).content.accounts
+
+    const foundAccount = accounts.find((account) => account === name)
+
+    if (foundAccount) {
+      throw new Error(`Account with name ${name} already exists`)
+    }
+
+    const foundIdentity = identities.find((identity) => identity.name === name)
+
+    if (foundIdentity) {
```

Comment:
nit - I actually thought it was kind of nice that you could use an existing identity

for example, if you create an identity with your ledger before trying to run this command then you won't be able to use it now